docs/developer/docs/source/markdown/forms/01-overview.md

```diff
@@ -0,0 +1,140 @@
+# Forms Module Overview
+
+## Purpose
+
+The Forms Module renders dynamic forms from a UI specification (`UISpec`) and handles data capture. It supports two operational modes: **editable** (data entry) and **view** (read-only display).
+
+## Architecture
+
+```mermaid
+graph TD
+    subgraph FormManagers
+        EFM[EditableFormManager]
+        PFM[PreviewFormManager]
+    end
+
+    subgraph FormManager
+        TS[TabbedSectionDisplay]
+        IS[InlineSectionDisplay]
+        FS[FormSection]
+        F[Field]
+    end
+
+    FR[Field Registry]
+
+    EFM --> FM[FormManager base]
+    PFM --> FM
+    FM --> TS
+    FM --> IS
+    TS --> FS
+    IS --> FS
+    FS --> F
+    FR -.->|resolves component| F
+```
+
+## Key Dependencies
+
+| Library                 | Purpose                                                  |
+| ----------------------- | -------------------------------------------------------- |
+| `@tanstack/react-form`  | Form state management, field binding, change tracking    |
+| `@tanstack/react-query` | Async data fetching, caching, mutations                  |
+| `zod`                   | Schema validation for field props and data               |
+| `@mui/material`         | UI components                                            |
+| `@faims3/data-model`    | Shared types, UISpec definitions, data engine interfaces |
+
+## Module Structure
+
+```
+lib/
+├── formModule/
+│   ├── formManagers/       # EditableFormManager, PreviewFormManager, FormManager
+│   ├── sections/           # TabbedSections, InlineSections, FormSection
+│   ├── Field.tsx           # Field renderer (resolves from registry)
+│   ├── Annotation.tsx      # Field annotation UI
+│   └── types.ts            # FaimsForm, FieldSpecification types
+├── fieldRegistry/
+│   ├── registry.ts         # Field type registry
+│   ├── fields/             # Individual field implementations
+│   └── types.ts            # FieldInfo, FieldReturnType
+├── rendering/
+│   ├── DataView.tsx        # Read-only form renderer
+│   └── fields/view/        # View-mode field renderers
+├── validationModule/
+│   └── validation.ts       # Zod schema compilation, validation utilities
+└── hooks/
+    ├── useAttachment.ts    # Attachment loading hook
+    └── useElementWidth.ts  # Responsive layout hook
+```
+
+## Data Flow
+
+### Edit Mode
+
+```mermaid
```

Comment:
I don't find this very useful as a summary - one long flowchart could easily be a bullet list